### PR TITLE
Add flipoff (inverted cycle) option - turn on and off

### PR DESCRIPTION
**Pull request by mishabunte**
I need a feature to turn the specified port on and off after a specific delay,

The example below is to turn the port #1 at location 1-1.4 on and to turn it off after 1.5 second delay

```bash
uhubctl -d 1.5 -a flipoff -l 1-1.4 -p 1
```

### CODE REVIEW

File: uhubctl.c [hunk 1]
```diff
@@ -1226,7 +1230,7 @@ int main(int argc, char *argv[])
             if (rc == 0) {
                 /* will operate on these ports */
                 int ports = ((1 << hubs[i].nports) - 1) & opt_ports;
-                int should_be_on = k;
+                int should_be_on = opt_action != POWER_FLIPOFF ? k : !k;
```
Comment (mvp) [inline uhubctl.c:1233]:
```
int should_be_on = k;
if (opt_action == POWER_FLASH) {
  k = !k;
}
```

**Review by mvp — COMMENTED** [review]
Thanks for your submission @mishabunte!

Flipoff doesn't sound great as a term. Can you please use `flash` instead? Usually flash means short power pulse or pulse of light, which is close to action you are proposing: turn on then quickly off.

Let me know if you want to submit a fix for this in this PR, or I can do it myself.

I'm planning to release new version 2.6 after merging this improvement.

Thanks a lot!

**Comment by mishabunte** [comment]
Thanks @mvp , flash sounds better. I've done the re-naming.

**Review by mvp — APPROVED** [review]
Thank you!

**Pull request merged**